apps/server/src/models/dataModel.ts

```diff
@@ -20,6 +20,11 @@ export const dbModel: DatabaseModel = {
   },
   viewSettings: {
     overrideStyles: false,
+    normalColor: '#ffffffcc',
+    warningColor: '#FFAB33',
+    warningThreshold: 3,
+    dangerColor: '#ED3333',
+    dangerThreshold: 1,
```

Comment:
is this supposed to be milliseconds?

Reply:
good catch. I think thats also why it didnt work for you in the beginning